admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/domain/service/AuthenticationService.kt

```diff
@@ -0,0 +1,18 @@
+package net.thechance.mena.admin_panel.domain.service
+
+import net.thechance.mena.admin_panel.domain.repository.AdminAuthenticationRepository
+import org.koin.core.annotation.Single
+
+@Single
+class AuthenticationService(
+    private val adminAuthenticationRepository:
+    AdminAuthenticationRepository
+) {
+    suspend fun getAccessToken(): String {
+        return adminAuthenticationRepository.getAccessToken()
+    }
```

Comment:
can here be like this 
```suggestion
    suspend fun getAccessToken(): String = adminAuthenticationRepository.getAccessToken()
    
```